libsignal-service/src/sender.rs

```diff
@@ -461,21 +477,11 @@ where
                             "dropping session with device {}",
                             extra_device_id
                         );
-                        if let Some(ref uuid) = recipient.uuid {
-                            self.cipher.store_context.delete_session(
-                                &libsignal_protocol::Address::new(
-                                    uuid.to_string(),
-                                    *extra_device_id,
-                                ),
-                            )?;
+                        if let Some(_uuid) = recipient.uuid {
+                            unimplemented!("deleting session: unimplemented following the switch to the Rust version of libsignal-protocol");
                         }
-                        if let Some(e164) = recipient.e164() {
-                            self.cipher.store_context.delete_session(
-                                &libsignal_protocol::Address::new(
-                                    &e164,
-                                    *extra_device_id,
-                                ),
-                            )?;
+                        if let Some(_e164) = recipient.e164() {
+                            unimplemented!("deleting session: unimplemented following the switch to the Rust version of libsignal-protocol");
```

Comment:
Tracking issue for this? This definitely affects Whisperfish.

Reply:
Weird, I swear I reverted that. I added `SessionStoreExt` so that's possible again.